dspace-spring-rest/src/test/java/org/dspace/app/rest/CommunityRestRepositoryIT.java

```diff
@@ -249,35 +248,74 @@ public void findAllSubCommunities() throws Exception{
                 .withLogo("SomeTest")
                 .build();
 
-        Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+        Community parentCommunityChild1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
                 .withName("Sub Community")
                 .build();
 
-        Community child12 = CommunityBuilder.createSubCommunity(context, child1)
+        Community parentCommunityChild2 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+                .withName("Sub Community2")
+                .build();
+
+        Community parentCommunityChild2Child1 = CommunityBuilder.createSubCommunity(context, parentCommunityChild2)
                 .withName("Sub Sub Community")
                 .build();
 
-        Community child2 = CommunityBuilder.createSubCommunity(context, parentCommunity2)
+
+        Community parentCommunity2Child1 = CommunityBuilder.createSubCommunity(context, parentCommunity2)
                 .withName("Sub2 Community")
                 .build();
 
-        Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
+        Collection col1 = CollectionBuilder.createCollection(context, parentCommunityChild1).withName("Collection 1").build();
 
-        getClient().perform(get("/api/core/communities/search/subCommunities/" + parentCommunity.getID()))
+        getClient().perform(get("/api/core/communities/search/subCommunities/")
```

Comment:
this is unexpected... the URL should be /api/core/communities/search/subCommunities without the final /
should we reject such request?